source/Octopus.Client/Model/DeploymentProcessResource.cs

```diff
@@ -52,13 +61,13 @@ public DeploymentStepResource AddOrUpdateStep(string name)
             return step;
         }
 
-        public DeploymentProcessResource RemoveStep(string name)
+        public DeploymentProcessBaseResource RemoveStep(string name)
```

Comment:
This signature shouldn't change, so push these methods down?

Reply:
These methods are shared for deployment steps and ops runs though, so doesn't it make sense for these to belong in the base class?